src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -470,58 +477,48 @@ class PartialUpdateSpec extends HealthCheckSpecification {
     def "Partial update with empty body does not actually update flow in any way(v1)"() {
         given: "A flow"
         def swPair = switchPairs.all().random()
-        def flow = flowHelperV2.randomFlow(swPair)
-        flowHelperV2.addFlow(flow)
-        def originalCookies = northbound.getSwitchRules(swPair.src.dpId).flowEntries.findAll {
+        def flow = flowFactory.getRandom(swPair)
+        def originalCookies = switchRulesFactory.get(swPair.src.dpId).getRules().findAll {
             def cookie = new Cookie(it.cookie)
             !cookie.serviceFlag || cookie.type == CookieType.MULTI_TABLE_INGRESS_RULES
         }*.cookie
 
         when: "Request a flow partial update without specifying any fields"
-        def flowBeforeUpdate = northboundV2.getFlow(flow.flowId)
-        northbound.partialUpdate(flow.flowId, new FlowPatchDto())
+        def flowBeforeUpdate = flow.retrieveDetails().convertToFlowPayload()
+        flow.partialUpdateV1(new FlowPatchDto())
 
         then: "Flow is left intact"
-        expect northboundV2.getFlow(flow.flowId), sameBeanAs(flowBeforeUpdate)
```

Comment:
Let's use our existing method in FlowExtended hasTheSamePropertiesAs()
 flow.partialUpdateV1(new FlowPatchDto())

then:
flow.retrieveDetails().hasTheSamePropertiesAs(flow)

Reply:
Done